### ISSUE TITLE
Improve deployment instructions

### ISSUE THREAD

**Issue by DaanVanVugt** [DaanVanVugt/Taxila#26]
Things I've run into:

```bash
root@elixirtess:/data/volume_2/TeSS# docker-compose -f docker-compose-prod.yml up -d --build
WARNING: The HOSTNAME variable is not set. Defaulting to a blank string.
```

-> `export HOSTNAME=staging.taxila.nl`

**Comment by DaanVanVugt** [comment]
Topics to take into account:

- Fixed IP in researchcloud

- [ ] Ansible script
  - [ ] create docker-capable host
  - [ ] clone our repository
  - [ ] create .env-production file
  - [ ] set HOSTNAME env var
  - [ ] run docker-compose with production settings

**Comment by eddytimmersurf** [comment]
Fixed IP for staging: 145.38.204.184
default deployment has been made and still needs to be verified

**Comment by DaanVanVugt** [comment]
See https://github.com/ElixirTeSS/TeSS/blob/master/docs/docker.md for a detailed docker setup guide
